buildspec.yml

```diff
@@ -0,0 +1,25 @@
+version: 0.2
+
+batch:
+  fast-fail: false
+  build-list:
+    - identifier: openjdk8
```

Comment:
For discussion: with Travis we were also checking openjdk9 and openjdk10. Do we want to bring them in here as well?

Reply:
I think that this is a good callout.
I would like to get this merged as it is additive right now.